nps-unit-sample.json

```diff
@@ -1,367 +1,367 @@
-                      "value": 11
-                    },
-                    {
-                      "year": 2002,
-                      "value": 15
-                    }
-                  ]
-                },
-                {
-                  "label": "Fine Sea Salt",
-                  "annualValues": [
-                    {
-                      "year": 2000,
-                      "value": 20
-                    },
-                    {
-                      "year": 2001,
-                      "value": 25
-                    },
-                    {
-                      "year": 2002,
-                      "value": 17
-                    }
-                  ]
-                },
-                {
-                  "label": "Fine Soil",
-                  "annualValues": [
-                    {
-                      "year": 2000,
-                      "value": 0.5
-                    },
-                    {
-                      "year": 2001,
-                      "value": 0.75
-                    },
-                    {
-                      "year": 2002,
-                      "value": 0.20
-                    }
-                  ]
-                }
-              ]
-            },
-            {
-              "name": "Haziest Days by Year",
-              "yAxisLabel": "Light Extinction, 1/Mm",
-              "variableLines": [
-                {
-                  "label": "Coarse Mass",
-                  "annualValues": [
-                    {
-                      "year": 2000,
-                      "value": 10
-                    },
-                    {
-                      "year": 2001,
-                      "value": 11
-                    },
-                    {
-                      "year": 2002,
-                      "value": 15
-                    }
-                  ]
-                },
-                {
-                  "label": "Fine Sea Salt",
-                  "annualValues": [
-                    {
-                      "year": 2000,
-                      "value": 20
-                    },
-                    {
-                      "year": 2001,
-                      "value": 25
-                    },
-                    {
-                      "year": 2002,
-                      "value": 17
-                    }
-                  ]
-                },
-                {
-                  "label": "Fine Soil",
-                  "annualValues": [
-                    {
-                      "year": 2000,
-                      "value": 0.5
-                    },
-                    {
-                      "year": 2001,
-                      "value": 0.75
-                    },
-                    {
-                      "year": 2002,
-                      "value": 0.20
-                    }
-                  ]
-                }
-              ]
-            }
-          ]
-        }
-      ]
-    },
-    "ozone": {
```

Comment:
repeat the above approach seen in `visibility` for the other params...

As other issues present themselves (different chart types or something) we can address them case-by-case, but generally if we follow the park > param > monitoring site > year data grouped into individual chart objects the front end will be **considerably** more straightforward.

Reply:
Okay.